sd/zk/integration_test.go

```diff
@@ -180,15 +177,29 @@ func TestGetEntriesPayloadOnServer(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	_, err = c.(*client).Create(
-		path+"/instance3",
-		[]byte("just some payload"),
-		stdzk.FlagEphemeral|stdzk.FlagSequence,
-		stdzk.WorldACL(stdzk.PermAll),
-	)
+
+	instance3 := Service{
+		Path: path,
+		Name: "instance3",
+		Data: []byte("just some payload"),
+	}
+	registrar := NewRegistrar(c, instance3, logger)
+	registrar.Register()
+	select {
+	case event := <-eventc:
+		if want, have := stdzk.EventNodeChildrenChanged.String(), event.Type.String(); want != have {
+			t.Errorf("want %s, have %s", want, have)
+		}
+	case <-time.After(20 * time.Millisecond):
```

Comment:
Experience dictates this will be flaky on Travis; can you make the minimum ca. 100ms?